diff --git a/typo3/sysext/core/Classes/Site/Entity/SiteLanguage.php b/typo3/sysext/core/Classes/Site/Entity/SiteLanguage.php
--- a/typo3/sysext/core/Classes/Site/Entity/SiteLanguage.php
+++ b/typo3/sysext/core/Classes/Site/Entity/SiteLanguage.php	(date 1743412799793)
@@ -181,6 +181,14 @@
         }
     }

+    /**
+     * Patched via belsignum/languagemodes adding support for v10 to change FallbackType dynamically
+     */
+    public function setFallbackType(string $fallbackType): void
+    {
+        $this->fallbackType = $fallbackType;
+    }
+
     /**
      * Returns the SiteLanguage in an array representation for e.g. the usage
      * in TypoScript.
